### PR TITLE
REPENT

### PR DESCRIPTION
**Pull request by WWWWW-Git**
## About The Pull Request

Have YOU ever wanted to be Mozgus?
o8cress5qdr31
avo2ib94z3881
Then boy have I got the PR for you!!!

The Preacher is a new Pilgrim role, and mainly RP focused in how it works. In essence, the Preacher is just that, a travelling Preacher who came to Rockhill with nothing more than what he has on him, which includes a handheld church bell, and THE BOOK Psydon version. The Preacher's main goal is to convert people, how they go about this can be entirely up to the player, being far more secretive and subversive in how they go about things, or, if they feel bold enough, doing it in a far more fanatical way.

This PR also fixes the priest undergarments not having sleevesprites for some reason.

## Why It's Good For The Game

Its just an overall minor role, depending on how the round goes the threat of a preacher to both the clergy and the keep could be an issue, and a point of conflict that creates more RP.

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/clothing/rogueclothes/shirts.dm [hunk 1]
```diff
@@ -31,9 +31,7 @@
 	name = "undervestments"
 	desc = ""
 	icon_state = "priestunder"
-	sleeved = null
-	allowed_sex = list(MALE)
-	allowed_race = list("human", "tiefling", "aasimar")
```
Comment (BadAtThisGame302) [inline code/modules/clothing/rogueclothes/shirts.dm:36]: Why remove the gender/race blockers?

File: code/game/objects/items/rogueitems/books.dm [hunk 2]
```diff
@@ -650,3 +650,62 @@ ____________End of Example*/
...
+
+/datum/status_effect/buff/blessed
+	id = "blessed"
+	alert_type = /atom/movable/screen/alert/status_effect/buff/blessed
+	effectedstats = list("fortune" = 1)
+	duration = 20 MINUTES
+
+/atom/movable/screen/alert/status_effect/buff/blessed
+	name = "Blessed"
+	desc = ""
```
Comment (BadAtThisGame302) [inline code/game/objects/items/rogueitems/books.dm:698]: Maybe add a desc like Psydon blesses you.

**Comment by Mogelix** [comment]
THIS IS SO AWESOME THIS IS SO AWESOME THIS IS SO AWESOME THIS IS SO AWESOME THIS IS SO AWESOME THIS IS SO AWESOME

**Comment by ThePainkiller** [comment]
# BASED CONTRIBUTION SPOTTED

File: code/modules/clothing/rogueclothes/shirts.dm (hunk 1 above)
Comment (ThePainkiller) [reply code/modules/clothing/rogueclothes/shirts.dm:36]: While that might be true for the sprites to be there, I wish this doesn't turn a justification to open the job to races that should not be playing the role itself

File: code/game/objects/items/rogueitems/books.dm (hunk 2 above)
Comment (Ickysoup) [reply code/game/objects/items/rogueitems/books.dm]: Isnt it illegal to have any mentions of Psydon in the code?

File: strings/psybibble.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+PSY 1:1... In the beginning there was only darkness, and so He made light.
+PSY 1:2... There was only light, and so He made Grimoria. And in Grimoria he made man in His own image.
+PSY 2:15... Astrata and Noc, His own children fought over who would have domain of the sun. Astrata defeated her brother, and in an act of spite, did he make nite.
+PSY 8:3... Sun and Moon, Daughter and Son both should be revered by man, but make no mistake that their creator is most deserving of worship. He who made all things. Blessed be His forgotten name.
+PSY 7:1... The Father of Men, Destroyer of the Wicked, The First Hero, all titles He was mantled with, and all titles every man should bow in respect of.
+PSY 9:7... The Elves, atrocities uncountable, were punished by Him for their hubris. Their immortality which they clung to plucked from them as one would pluck a thorn from ones side.
```
Comment (ThePainkiller) [inline strings/psybibble.txt:6]: How ironic considering Psydon himself IS an elf 🙂

Comment (sanshoom) [inline strings/psybibble.txt:6]: 14:88 Kill elves HORRIBLY with STONES!

File: code/modules/clothing/rogueclothes/shirts.dm (hunk 1 above)
Comment (BadAtThisGame302) [reply code/modules/clothing/rogueclothes/shirts.dm:36]: Yes but there can be Female/Aasimar Priests

**Review by Optimism333 — COMMENTED** [review]
Haven't seen any opposition really. Another Weeping God worshipping roll makes me roll my eyes, but it seems popular, so I'll move to approve it within the day if no major objections are raised.

**Review by BadAtThisGame302 — APPROVED** [review]

**Comment by ThePainkiller** [comment]
I know people hate so much psydon love but I trust this will be epic because it has zero combat skills and will most likely be a fluff role played to be town preacher and buffer (we need more buff-based classes and less combat roles)

image

**Review by Optimism333 — COMMENTED** [review]
Merge might have to wait until #949 gets out of testing.

After that I'll expedite this.

**Review by Optimism333 — APPROVED** [review]
I'll approve this, but with the caveat that I'd like future religious additions to focus more on content for the church of the Ten. We're starting to overload on Psydon worshipping roles, in my opinion.

**Pull request merged**